src/match/tasks.py

```diff
@@ -63,13 +61,9 @@ def matcher():
             print(f"Match error between {pear}: {error_msg}")
 
 
-schedule, _ = IntervalSchedule.objects.get_or_create(
-    every=1,
-    period=IntervalSchedule.MINUTES,
-)
-
-PeriodicTask.objects.get_or_create(
-    interval=schedule,
-    name="Pear Algorithm",
-    task="match.tasks.matcher",
-)
+# TODO: uncomment after proof of concept works on dev server
+# PeriodicTask.objects.get_or_create(
```

Comment:
make sure that you delete this in the Postgres database as well, otherwise it'll continue to run.